sidecar/app/query/ipa.py

```diff
@@ -261,17 +272,11 @@ def run(self):
             )
             while True:
                 r = httpx.get(url).json()
-                status = r.get("status")
+                status = Status.from_json(r)
```

Comment:
if the status doesn't match, won't we be having an infinite loop here?

Reply:
yes, that's intentional. if there isn't yet a status from the other machines, we wait until there is one (and it's running.) if something causes it to never resolve, we have the ability to kill manually.

we could add a max timeout that would cause it to auto crash.

Reply:
ok, added a thing here so that if it's unknown for a long time, it crashes